addons/website_sale/controllers/main.py

```diff
@@ -152,11 +153,11 @@ def _get_pricelist_context(self):
 
         return pricelist_context, pricelist
 
-    def _get_compute_currency(self, pricelist, product=None):
```

Comment:
You change the signature of this function (I am not sure why) and don't adapt the multiple occurence I have in my code, what am I missing ?

Reply:
I changed the signature because I need to compute currency from pricelist to pricelist, from pricelist to product and from product to pricelist.